odoo/tools/pycompat.py

```diff
@@ -23,8 +24,16 @@ def implements_iterator(cls):
 
     exec ('def reraise(tp, value, tb=None):\n raise tp, value, tb')
 else:
+    import builtins, math
     # pylint: disable=bad-functions
     integer_types = (int,)
+    def round(f):
+        if builtins.round(f + 1) - builtins.round(f) != 1:
+            return f + abs(f) / f * 0.5
```

Comment:
That seems to make sense. I got this from THE INTERNET and given that it seemed to work (aside from the next section which I commented which didn't handle -0 correctly) I didn't go further.